HW02_A_Graph-Fails.Rmd

```diff
@@ -35,17 +35,32 @@ data(mpg) #this is a dataset from the ggplot2 package
 mpg %>% 
   ggplot(mapping = aes(x = city, y = hwy, color = "blue")) %>% 
   geom_point()
+#The issue here arises, as the error message explains, due to the use of the pipe operator " %>%  " instead of +
+#The pipe operator would place the result of the ggplot() function inside of geom_plot() function, which R cannot understand

```

Comment:
This is a really nice explanation of how the pipe operator is interpreting the code. A small clarification: did you mean to say the pipe operator would place the result into geom_point() instead of geom_plot()?

Reply:
I did mean that!